src/main/java/com/salesforce/dev/pages/Home/HomePage.java

```diff
@@ -20,6 +21,7 @@
 
    private WebDriver driver;
    private WebDriverWait wait;
+   private CommonOperation operation = new CommonOperation();
```

Comment:
you don't need to instance this class
because you have public static methods
you can call them directly